observatory-platform/observatory/platform/telescopes/templates/dag.py.jinja2

```diff
@@ -12,11 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # Author: Tuan Chien #}
+# Copyright 2021 Curtin University
```

Comment:
Not sure if we should be specifying the Curtin copyright in the templates.  I'm not exactly sure who the likely users of the platform are, but it's perhaps better to leave any copyright notices for people to set themselves.

Reply:
Yeah good point, would you suggest just removing this specific line but leaving the remainder of the license in?
I think it would be good to ensure that people add a license to their file.

Reply:
Sounds good, I have added some info on this in the section on the `generate workflow` command